src/components/team/AddMember.vue

```diff
@@ -12,27 +12,25 @@
         :key="index"
         :class="['tag', 'is-medium', isSelected(attribute) ? 'is-primary' : '']"
         @click="addAttribute(attribute)"
-      >
-        {{ attribute.name }}
-      </span>
+      >{{ attribute.name }}</span>
     </div>
     <div class="control">
-      <button class="button is-inverted is-outlined" @click="addMember">
-        Teammitglied hinzufügen
-      </button>
+      <button class="button is-inverted is-outlined" @click="addMember">Teammitglied hinzufügen</button>
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from "vue-property-decorator";
 import { Member, PlayerAttribute } from "@/types";
-import { Action, State } from "vuex-class";
+import { namespace } from "vuex-class";
+
+const teamModule = namespace("@/store/team.ts");
```

Comment:
@markusphil this doesn't seem to work, telling me it can't find that file (also i think it's weird to include it like that